packages/workflow/test/Helpers.ts

```diff
@@ -109,6 +107,7 @@ class NodeTypesClass implements INodeTypes {
 
 let nodeTypesInstance: NodeTypesClass | undefined;
 
+// eslint-disable-next-line
```

Comment:
Which specific rule are we disabling here? And why?

Reply:
the function following this line is named `NodeTypes` which is against our naming convention of camelCase for functions